app/stylesheets/base/_ic_app_layout.scss

```diff
@@ -249,12 +249,124 @@ body:not(.course-menu-expanded) {
   }
 }
 
+.sm-course-menu.list-view{
+  padding: 0 10px;
+  position: relative;
+  border-bottom: $sm-gray-4;
```

Comment:
I'm quite nervous (though not necessarily a complete block) on using variable names that reflect an actual style rather than a purpose. I'm particularly afraid that this will become a pattern (though I understand your need for a temporary fix).